BridgeAppSDK/SBABridgeManager.h

```diff
@@ -120,6 +120,11 @@ typedef void (^SBABridgeManagerCompletionBlock)(id _Nullable responseObject, NSE
     completion:(SBABridgeManagerCompletionBlock _Nullable)completionBlock;
 
 /*!
+ Sign out user on the bridge server
```

Comment:
missing '.' at the end, Bridge should be capitalized, and should document the completion parameter (see above).